src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const dateArray = date.split(`${fromFormat[3]}`);
+  const givenDate = {
+    [fromFormat[0]]: dateArray[0],
+    [fromFormat[1]]: dateArray[1],
+    [fromFormat[2]]: dateArray[2],
+  };
+  const dateInNewFormat = [];
+
+  if ('YY' in givenDate) {
+    if (givenDate.YY < 23) {
```

Comment:
Try to avoid nested if/else statements

https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240

Reply:
Thank you for the code review and for the needest information! Fixed.